apps/condo/domains/billing/utils/countrySpecificQRCodeParsers/ru.qrCodeParser.js

```diff
@@ -14,26 +17,26 @@ const { get } = require('lodash')
 
 /**
  * @param {string} qrStr The QR code string got from the picture
+ * @param {function} logger ValidateQRCodeService logger
  * @return {TRUQRCodeFields}
  */
 function parseRUReceiptQRCode (qrStr) {
-    const matches = /^ST(?<version>\d{4})(?<encodingTag>\d)\|(?<requisitesStr>.*)$/g.exec(qrStr)
+    const buffer = Buffer.from(qrStr, 'base64')
+    const detectedEncoding = detectEncoding(buffer)
+    const decodedRequisitesStr = convertEncoding(buffer, detectedEncoding)
```

Comment:
This is no requisites string. This is all qr-code content
```suggestion
    const decodedQrStr = convertEncoding(buffer, detectedEncoding)
```